### ISSUE TITLE
Platform description dropdown, values from NERC L06

### ISSUE THREAD

**Issue by n-a-t-e** [cioos-siooc/metadata-entry-form#179]
Platform description could use an [NERC L06 Vocabulary](http://vocab.nerc.ac.uk/collection/L06/current/) dropdown for the description, and using L06 is recommended by the CIOOS profile. This will be helpful for filtering datasets in CDE.

The CIOOS profile suggests for this field:
_description of the platform or platform type. Recommend including NERC L06 term and description (example: Research vessel  - A platform of any size operating on the surface of the water column in unpredictable locations that is specifically equipped, manned and operated for scientific, usually oceanographic, research.)_

Since we are already populating the platform description with free text, there would be a question of how to combine the free text and the L06 code. They could be combined, like:

```xml
<mcc:description>
  <gco:CharacterString>
  Moored surface buoy - An  unmanned instrumented platform operating on the surface of the water  column loosely tethered to the seafloor to maintain a fixed position  (e.g. ODAS buoy).
  
  Surface Mooring with a simple 30lb boat anchor. Instrumentation is fix to an
  auxiliary line which is suspended from the surface buoy and along the main anchor line.
  </gco:CharacterString>
</mcc:description>
```

**Comment by n-a-t-e** [comment]
We can use the `otherProperty` field to get the NERC L06 vocabulary in. `otherProperty` being a field that "_defines attributes not explicitly included in MI_Platform_".  Another option is the keywords section, with "SeaVoX Platform Categories" (NERC L06) as its own thesaurus. 

```xml
</mac:instrument>
<mac:otherProperty xlink:href="http://vocab.nerc.ac.uk/collection/L06/current/" xlink:title="SeaVoX Platform Categories">
    <gco:Record>
        <gco:CharacterString>drifting subsurface profiling float</gco:CharacterString>
    </gco:Record>
</mac:otherProperty>
```

**Comment by n-a-t-e** [comment]
Fixed by #203
